name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/csv/RentabloNetExporter.java

```diff
@@ -62,18 +64,21 @@ private void writeDividend(CSVPrinter printer, AccountTransaction transaction) t
         if (transaction.getSecurity() == null)
             return;
         if (transaction.getType() != AccountTransaction.Type.INTEREST
+                        && transaction.getType() != AccountTransaction.Type.INTEREST_CHARGE
                         && transaction.getType() != AccountTransaction.Type.DIVIDENDS)
             return;
 
         printer.print(DATE_FORMAT.format(transaction.getDateTime().toLocalDate()));
         printer.print(CSVExporter.escapeNull(transaction.getSecurity().getIsin()));
         printer.print(CSVExporter.escapeNull(transaction.getSecurity().getName()));
-        printer.print("Aktie");
```

Comment:
Did you test the Rentablo import with foreign currencies?
Because back then, the column really had to be labeled "Aktie".
I am not sure if the Rantable would recognize our translations.

Reply:
Hello @buchen 
I have sent an email request to Rentablo.de to clarify this. Also I have asked to tell me if anything has changed in the structure of the CSV file.
In the meantime, I am waiting for a reply and Rentablo.de has opened a support ticket.

Since I thought that the type "Aktie" is a fixed column, I have given this a fixed label. (German translation)

Alex

Reply:
> Since I thought that the type "Aktie" is a fixed column, I have given this a fixed label. (German translation)

?? This PR removes that static label, and replaces it with the translated property

```
printer.print(Messages.CSVColumn_Security);
```